people/JamesPatrickGill.toml

```diff
@@ -0,0 +1,3 @@
+name = 'James Gill'
+github = 'JamesPatrickGill'
+github-id = 44863195
```

Comment:
You might want to add your Zulip ID here. I believe it's not used yet but it might be in the future

Reply:
Is this my actual Zulip id? Can't find it haha